src/main/scala/sigmastate/lang/SigmaBuilder.scala

```diff
@@ -147,6 +146,10 @@ trait SigmaBuilder {
   def mkBase64ToByteArray(input: Value[SString.type]): Value[SByteArray]
 
   def mkPK(input: Value[SString.type]): Value[SSigmaProp.type]
+  def mkGetVar[T <: SType](varId: Byte, tpe: SOption[T]): Value[SType]
```

Comment:
@greenhat Why not `def mkGetVar[T <: SType](varId: Byte, tpe: T): Value[SOption[T]]`?

Reply:
@ergomorphic Good catch! Thanks. Makes much more sense. Also synced `mkGetVar` and `apply` behavior  (wrap `tpe` in `SOption`).